addon/bootstrap.js

```diff
@@ -40,6 +40,7 @@ const REASONS = {
   ADDON_UPGRADE: 7,    // The add-on is being upgraded.
   ADDON_DOWNGRADE: 8,  // The add-on is being downgraded.
 };
+
 for (const r in REASONS) { REASONS[REASONS[r]] = r; }
```

Comment:
It's better not to use `console.log`, but rather rely on the built-in logging facility that can be easily turned off/on without cluttering the user console. Anyway, this doesn't answer the "why is this needed" question :)